### ISSUE TITLE
Complete warnings/shims for jQuery 1.8 deprecations

### ISSUE THREAD

**Issue by dmethvin** [jquery/jquery-migrate#89]
http://api.jquery.com/category/deprecated/deprecated-1.8/

- [x] Docs in warnings.md for `.andSelf()` and `.size()` (Ref #62)
- [x] `.load()`
- [x] `.unload()`
- [x] `deferred.pipe()`

N.B. Since the `.load()` and `.unload()` shims are just like [`.error()`](https://github.com/jquery/jquery-migrate/blob/683b9b3ae40e3fe9e83f3dd39d8e5e958975f3fa/src/event.js#L41-51) the existing code can just be generalized. None of those need to bubble so `.triggerHandler()` will do the trick.

~~@gibson042 Now that we're going all Promise/A+ on Deferred should we un-deprecate `deferred.pipe` to preserve a way to get the old behavior? If so I can open a docs ticket for that tagged for 3.0.~~

**Comment by dmethvin** [comment]
As discussed in the meeting today, we're leaving `deferred.pipe()` deprecated and should warn for it.

**Comment by dmethvin** [comment]
I'll close once the 3 PRs land

**Comment by dmethvin** [comment]
All related PRs have landed. :fireworks:
